packages/core/src/components/Button/Button.example.jsx

```diff
@@ -22,10 +23,14 @@ ReactDOM.render(
     <Button className="ds-u-margin-right--1" disabled>
       Button with `disabled` prop
     </Button>
-    <Button className="ds-u-margin-right--1" variation="transparent" href="javascript:void(0);">
+    <Button
+      className="ds-u-margin-right--1"
+      href="javascript:void(0);"
+      onClick={() => alert('Link styled as button clicked!')}
+    >
       Button with `href` prop
     </Button>
-    <Button className="ds-u-margin-right--1" component={Link}>
+    <Button className="ds-u-margin-right--1" component={Link} href="/">
```

Comment:
I'm wondering if we could leave the `<button>` as a `<button>` here instead of switching it to a link. Generally, I think we should use links for navigating and not triggering functionality on the page.

Reply:
Its the only way to keep the buttons in the legend and keep the existing design
https://adhoc.slack.com/archives/C645GLLNN/p1583190102065800?thread_ts=1582927324.039200&cid=C645GLLNN

Reply:
I was thinking that since the buttons are not inside the legend that we could do something like what Trevor suggested from that thread: 

> You might try fieldset, legend, div with buttons, div with checkboxes + labels, and use aria-describedby to reference the legend on each checkbox.

Reply:
I see what you mean, the last commit implements that